README.md

```diff
@@ -0,0 +1,8 @@
+# Storefront build configuration for Shopware
+
+This plugin allows building Shopware storefronts without accessing the database,
+e.g. during container image builds.
```

Comment:
We need a bit more detailed README, explaining what the plugins does, i.e:
* Ensures the plugins config and the theme config are loaded from disk
* Changes the location of the theme config files from `files/` to `var/` as the `files/` is commonly a shared volume that should not hold application-version specific configs.

Reply:
Added.